mypy/checkmember.py

```diff
@@ -383,7 +383,18 @@ def analyze_member_var_access(name: str,
           not mx.is_operator):
         if not mx.is_lvalue:
             for method_name in ('__getattribute__', '__getattr__'):
-                method = info.get_method(method_name)
+                method_node = info.get(method_name)
+
+                # We cannot use `.get_method()` here, because it only returns
```

Comment:
Would it be better to fix `get_method` to also work for decorated methods? The same issue could come up elsewhere.

Reply:
It is used in quite a lot of places right now:

```
» grep -o '.get_method(' -r mypy        
mypy/checkmember.py:.get_method(
mypy/checkmember.py:.get_method(
mypy/checkmember.py:.get_method(
mypy/checkmember.py:.get_method(
mypy/typeshed/stdlib/@python2/urllib2.pyi: get_method(
mypy/typeshed/stdlib/urllib/request.pyi: get_method(
mypy/nodes.py: get_method(
mypy/checker.py:.get_method(
mypy/checker.py:.get_method(
mypy/checker.py:.get_method(
mypy/subtypes.py:.get_method(
mypy/subtypes.py:.get_method(
mypy/subtypes.py:.get_method(
mypy/subtypes.py:.get_method(
```

We can probably ignore 2 `mypy/typeshed` usages, which gives us 12 total.
I will try to push an experimental change with re-worked `get_method()` to see what happens.

Reply:
A lot of tests fail locally for this change: https://github.com/python/mypy/pull/11150/commits/c9674863a6255d62ada03b0b87f945c5b52f4d72 Let's wait for the CI to finish.